src/client.py

```diff
@@ -142,7 +143,10 @@ def process_data(self, start_date=None, end_date=None):
                 try:
                     # Duplicate flagger requires a special call.
                     if flagger.name == "Duplicate":
-                        flags.update(flagger.flag(row_id, ctran_df))
+                        #flags.update(flagger.flag(row, ctran_df))
```

Comment:
Remove commented out code.

Reply:
This will mess up 1 other test (I believe #2), as it adds additional flag. Until duplicate is correctly implemented, it will keep failing (just fyi)